analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -288,7 +288,8 @@ abstract class PackageManager(val projectType: String) : Plugin {
 
         val result = mutableMapOf<File, List<ProjectAnalyzerResult>>()
 
-        definitionFiles.forEach { definitionFile ->
+        // Create a stable order to process the definition files, starting with the ones with the most path segments.
```

Comment:
commit-msg: Would you mind explaining _why_ this work around works / what the underlying issue is? (e.g. I'm after understanding which bits of the `Npm` analysis are depending on ordering of the definition files.